File: app/src/main/java/ru/job4j/todolist/store/TodoDbSchema.java
package ru.job4j.todolist.store;

public class TodoDbSchema {
    public static final class TodoTable {
        public static final String NAME = "todo";

        public static final class Cols {
            public static final String NAME = "name";
            public static final String DESC = "desc";
            public static final String CREATED = "created";
            public static final String CLOSED = "closed";
        }
    }
}

File: app/src/main/java/ru/job4j/todolist/store/TodoDbSchema.kt
package ru.job4j.todolist.store

class TodoDbSchema {
    object TodoTable {
        const val NAME = "todo"

        object Cols {
            const val NAME = "name"
            const val DESC = "desc"
            const val CREATED = "created"
            const val CLOSED = "closed"
            const val PHOTO = "photo"
        }
    }
}